### DISCUSSION TITLE
Vorta on Windows

### DISCUSSION THREAD

**Discussion by RayyanAnsari** [borgbase/vorta#1590] [Ideas]
Recently, I've been working on getting borg to run on Windows - most of the testsuite is now passing. I have gotten Vorta to build on Windows as well, and currently it can create repositories, but not add files to them (because of an error which I'm guessing is because of a change from borg v2 b1 and current master: `Options --progress and --list do not play nicely together.`).

![image](https://user-images.githubusercontent.com/68647953/217905365-7652fb0f-5cbe-45a3-82d2-18a7b1d65371.png)

My current changes can be found on this branch: https://github.com/RayyanAnsari/vorta/tree/windows

Would this be something that you would be interested in merging once it is able to perform basic actions? Borg on Windows is still in its early stages and has not been tested much, but this could be used as a base for proper support later.

**Comment by m3nu** [comment]
Yeah, would be cool. How do you run Borg on Windows?

**Reply by RayyanAnsari** [reply]
It's packaged in a standalone .exe file using pyinstaller.

![image](https://user-images.githubusercontent.com/68647953/217922901-19c8434f-c914-438b-9380-6c6443127e75.png)

[Here](https://github.com/borgbackup/borg/pulls?q=is%3Apr+is%3Aclosed+author%3ARayyanAnsari) are some of my PRs that have improved the support of Windows in Borg.

**Reply by m3nu** [reply]
That's great! If Thomas is doing an official Windows release, we should do one for Vorta (which also uses Pyinstaller).

**Reply by m3nu** [reply]
The reason being that we ship Borg with Vorta, so users only download one of them. Works well on macOS.

**Comment by RayyanAnsari** [comment]
https://user-images.githubusercontent.com/68647953/218160823-1c5414b4-151a-46c6-b44a-1e8a1fcbcd46.mp4

**Comment by apiening** [comment]
Looks interesting to me. Does the windows version make progress, or are there severe issues?
